### PR TITLE
[fastlane] Updated dependencies

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: fastlane/fastlane.gemspec [hunk 1]
```diff
@@ -33,11 +33,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'fastlane_core', '>= 0.37.0', '< 1.0.0' # all shared code and dependencies
```
Comment (i2amsam) [inline fastlane/fastlane.gemspec]: Should we bump this to 0.39.0 too?

Comment (KrauseFx) [inline fastlane/fastlane.gemspec]: Yeah, let's do that in a separate PR :+1:
